infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/fsm/TestDataSender.java

```diff
@@ -59,7 +64,13 @@ public TestDataSender() {
 
         return lastSentMessage;
     }
-    
+
+    @Override
+    public LogReplicationQueryMetadataResponse sendQueryMetadataRequest() throws ExecutionException, InterruptedException {
```

Comment:
Why do we have a Test class within infrastructure? Should this be in the test module?

Reply:
@annym , should we move this to other place?

Reply:
yes we can move this down... It's a sample DataSender exclusively for testing.